service-api/module/Application/src/Controller/IdentityController.php

```diff
@@ -564,4 +564,41 @@ public function updateDobAction(): JsonModel
 
         return new JsonModel($response);
     }
+
+    public function updateCpPoIdAction(): JsonModel
+    {
+        $uuid = $this->params()->fromRoute('uuid');
+        $data = json_decode($this->getRequest()->getContent(), true);
+        $response = [];
+        $status = Response::STATUS_CODE_200;
+
+        if (! $uuid) {
+            $status = Response::STATUS_CODE_400;
+            $this->getResponse()->setStatusCode($status);
+            $response = [
+                "error" => "Missing UUID"
+            ];
+            return new JsonModel($response);
+        }
+
+        try {
+            $this->dataImportHandler->updateCaseData(
+                $uuid,
+                'id_method_intl',
+                'M',
+                array_map(fn (mixed $v) => [
+                    'S' => $v
+                ], $data),
+            );
+        } catch (\Exception $exception) {
+            $response['result'] = "Not Updated";
+            $response['error'] = $exception->getMessage();
+            return new JsonModel($response);
```

Comment:
This should log the exception, set the response status to 500 and return `new Problem()` rather than  JsonModel.